src/simulator/world.py

```diff
@@ -135,8 +135,9 @@ def run(self) -> None:
             self.__num_evolutions += 1
 
         self.dump_all_shapes_info()
+        self.dump_all_owners_inf()
         self.dump_vis_data_to_file()
-
+    
```

Comment:
Styling issue. In future, please ensure that such trivial changes are not included in your commits.